script/01_DeployMockVeToken.s.sol

```diff
@@ -3,22 +3,36 @@ pragma solidity ^0.8.24;
 
 import "forge-std/Script.sol";
 import {BaseScript} from "./BaseScript.sol";
+import {Create3Factory} from "pancake-create3-factory/src/Create3Factory.sol";
 
 import {MockERC20} from "solmate/src/test/utils/mocks/MockERC20.sol";
 
 /**
+ * Step 1: Deploy
  * forge script script/01_DeployMockVeToken.s.sol:DeployMockVeTokenScript -vvv \
  *     --rpc-url $RPC_URL \
  *     --broadcast \
- *     --slow \
- *     --verify
+ *     --slow
+ *
+ * Step 2: Verify
+ * forge verify-contract <address> lib/pancake-v4-universal-router/lib/pancake-v4-periphery/lib/pancake-v4-core/lib/solmate/src/test/utils/mocks/MockERC20.sol:MockERC20 --watch \
+ *      --chain <chain_id> --constructor-args $(cast abi-encode "constructor(string, string, uint256)" "MockVeCake" "VeCake" "18")
  */
 contract DeployMockVeTokenScript is BaseScript {
+    function getDeploymentSalt() public pure override returns (bytes32) {
+        return keccak256("PANCAKE-V4-HOOKS/MockERC20/0.90");
```

Comment:
minor, can update next time, should be `MockVeCake` instead of `MockERC20`

Reply:
https://github.com/pancakeswap/pancake-v4-hooks/issues/22